styles.css

```diff
@@ -0,0 +1,113 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: montserrat;
+    color: #fff;
+}
+
+/* **Header** */
+header {
+    background-color: #46424D;
+    height: 60px;
+}
+h1 {
+    padding: 1% 0;
```

Comment:
Agreed to change from` %` since they're a little more responsive to parent element positioning and trickier to work with but produce similar results. I think some hacks would be okay up to the MVP (padding-bottom: 100%) but should be refactored past this point.
With respect to developer empathy (accessibility can get a little wonky with px), maybe we should discuss going with the `rem` or `em` (self-teaching on units still) for padding/sizing of elements?